src/presentation/controllers/configuration-controller.ts

```diff
@@ -0,0 +1,44 @@
+import { BaseController, GetRequest, RestController } from './base-controller'
+import { HttpErrorHandler } from '../interfaces/http-error-handler'
+import { Request, Response } from 'express'
+import { ConfigurationRepository } from '../../data-access/repositories/interfaces/configuration-repository'
+import { Exception } from '../../model/exceptions/exception'
+import { Configuration } from '../../model/entities/configuration'
+import { ShowStyleVariantRepository } from '../../data-access/repositories/interfaces/show-style-variant-repository'
+import { ShowStyleVariant } from '../../model/entities/show-style-variant'
+
+@RestController('/configurations')
+export class ConfigurationController extends BaseController {
+  constructor(
+    private readonly configurationRepository: ConfigurationRepository,
+    private readonly showStyleVariantRepository: ShowStyleVariantRepository,
+    private readonly httpErrorHandler: HttpErrorHandler
+  ) {
+    super()
+  }
+
+  @GetRequest()
+  public async getBlueprintConfiguration(_req: Request, res: Response): Promise<void> {
+    try {
+      const configuration: Configuration = await this.configurationRepository.getConfiguration()
+      res.send({
+        showStyleConfiguration: configuration.showStyle.blueprintConfiguration,
+        studioConfiguration: configuration.studio.blueprintConfiguration
+      })
+    } catch (error) {
+      this.httpErrorHandler.handleError(res, error as Exception)
+    }
+  }
+
+  @GetRequest('/:rundownId')
```

Comment:
This should be `'/rundowns/:rundownId'`.
If you leave it as `'/:rundownId'`, then the URL will look like this `'configurations/this-is-an-id'` where `this-is-an-id` refers to a Rundown id, but by only looking at the URL, REST says that is is a Configuration id, which it is not.

In REST, you must put the entity before the id in your URL, so if you want to add a Rundown id in your Configuration URL, the URL must look like `'configurations/rundowns/this-is-an-id'`.

The reasons that `RundownController` does not need to specify `/rundowns` in each endpoint is becasue `/rundowns` is specified on the controller level, just as you have speficied `/configurations` on the controller level in here.

Reply:
Fixed in next commit.